### ISSUE TITLE
Quais são as unidades de medidas suportadas na LIO

### ISSUE THREAD

**Issue by magnocosta** [DeveloperCielo/LIO-SDK-API-Integracao-Remota-v1-Java#5]
Bom dia,

Na documentação está escrito que os valores para unidade de medida são:

> EACH, HOURS, DAYS, SECONDS, CRATE_OF_12, SIX_PACK, GALLON e LITRE

Porem eu fico em duvida se estes valores são constantes, ou são somente exemplos, pois geralmente no mercado o pessoal usa unidades de medida como UN, CX, PCT e etc..

Caso sejam constantes, seria interessante contemplar as 'Tabela Padrão de Unidades de Medidas Comerciais' criado receita federal para uso na NF-e. Segue abaixo o link com a tabela:

http://www.nfe.fazenda.gov.br/portal/exibirArquivo.aspx?conteudo=RY8TPoGpyMY=

**Obs:**
Criado por @frekele na issue #3, eu apenas extrai o conteúdo para essa issue na tentativa de deixar mais claro e fácil para as pessoas com a mesma dúvida.

Abs!

**Comment by frekele** [comment]
Boa Noite,

Outro ponto relacionado é o tamanho do campo, em meus testes consigo persistir um valor aleatório de 500 caracteres ou mais no campo de unidade de medida e em outros.

Seria interessante posteriormente criar definições do tamanho máximo dos campos assim como temos no cielo e-commerce para garantir uma melhor consistência.

Sei que vocês devem estar tendo um grande trabalho, pois a Cielo lio é algo inovador que merece muitos elogios, quem teve a ideia merece muita parabenização, pois realmente facilitou muito a vida dos lojistas, principalmente os que não querem ou podem utilizar TEF.

Mas então, quando sobrar um tempo, quem sabe esta limitação de campos pode interessante.


Abraços

**Comment by magnocosta** [comment]
@frekele,

Eu acho muito válida a idéia de limitar as unidades de medidas, levando em consideração a tabela proposta. Nós estamos fazendo algumas melhorias no serviço de sincronização de ordens e acho que faz total sentido adicionar isso.

Eu sou apenas um dev e não tenho pode de priorizar as tarefas mas já criei a issue e adicionei a informação para atualizar aqui, quando ela for finalizada.

Segue o id da issue internamente, caso alguém do time volte aqui: LIO-4206

Abs!

**Comment by magnocosta** [comment]
Até lá vamos deixar essa issue como open mesmo.

**Comment by frekele** [comment]
Boa tarde @magnocosta ,

então eu devo considerar que estes campos não são constantes? Posso informar o valor que desejar?

**Comment by magnocosta** [comment]
@frekele,

Sim, hoje não existem limitações nesse campo. Esse é um campo texto de 255 caracteres que o usuário da API pode usar como quiser.
